extension/src/test/fixtures/expShow/sorted/tableData.ts

```diff
@@ -0,0 +1,19 @@
+import columns from './columns'
+import rows from './rows'
+import defaultData from '../base/tableData'
+import { TableData } from '../../../../experiments/webview/contract'
+
+const data: TableData = {
+  ...defaultData,
```

Comment:
`sorted/tableData` is a slightly simplified (no deps and no selected exps) version of `base/tableData`. We could simplify further though if preferred.